python/parser.js

```diff
@@ -25,7 +27,8 @@
     if (cs.sendDom !== undefined) options += `.send_dom(${serialize(cs.sendDom)})`
     if (cs.variationGroupId) options += `.variation_group_id(${serialize(cs.variationGroupId)})`
     if (cs.matchLevel) options += `.match_level(MatchLevel.${cs.matchLevel.toUpperCase()})`
-    if (cs.isFully) options += '.fully()'
+    if (cs.hooks) options += handleHooks(cs.hooks)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    if (cs.hooks) {options += handleHooks(cs.hooks)}
```